benchmarks/Smalltalk/build-image-squeak.st

```diff
@@ -0,0 +1,65 @@
+| parentDir runner loader starting |
+parentDir := (FileDirectory default fileNamed: Smalltalk documentPath)
+	directoryEntry containingDirectory.
+
+#('Scripting.st' 'SomLoader-Compiler.st' 'SomLoader-Tests.st') do: [:ea |
+	(parentDir fileNamed: ea) fileIn].
+
+Utilities setAuthorInitials: 'SomLoader'.
+
+ByteString compile: 'asFileReference
+    ^ FileDirectory default on: self' classified: '*SomLoader'.
+
+DosFileDirectory compile: 'entries
+    ^ super entries reject: [:e | e fullName includes: $#]' classified: '*SomLoader'.
+
+Behavior compile: 'subclass: a instanceVariableNames: b classVariableNames: c package: d
+    ^ self subclass: a instanceVariableNames: b classVariableNames: c poolDictionaries: '''' category: d' classified: '*SomLoader'.
+
+Character compile: 'canBeGlobalVarInitial
+    ^ self isLetter' classified: '*SomLoader-override'.
+
+Time class methodDict at: #primUTCMicrosecondsClock put: (Time class >> #utcMicrosecondClock).
+
+SmallInteger methodDict at: #'&' put: (SmallInteger >> #bitAnd:).
+SmallInteger methodDict at: #'%' put: (SmallInteger >> #\\).
+
+Smalltalk class compile: 'exit: code
+    ^ Smalltalk quitPrimitive' classified: '*SomLoader'.
+
+(Smalltalk classNamed: 'SomLoaderTests') compile: 'testLoadCDRedBlackTree
```

Comment:
Hm, the `FileSystem slash` stuff, could that perhaps better go into the original code instead of doing it after the fact?